CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 2.3.1
+
+### Patches
+* Use AWS-LC [v1.15.0](https://github.com/aws/aws-lc/releases/tag/v1.15.0) for ACCP
```

Comment:
do we need to update the submodule commit ID?

Reply:
Done. It turns out that AWS-LC now needs Go to build and our CI task for the legacy build did not have it.

Reply:
oh, that's annoying. thanks for resolving that issue!